lib/features/rules/BpmnRules.js

```diff
@@ -786,6 +786,10 @@ function canResize(shape, newBounds) {
     return true;
   }
 
+  if (isGroup(shape)) {
```

Comment:
Could we also add a test case that verifies that groups are properly resized in the BPMN 2.0 XML?